### PR TITLE
Piano pattern page

### PR DESCRIPTION
**Pull request by jamesjacobs**
This branch adds a patterns page for the piano layout.

### CODE REVIEW

**Comment by jackbentley** [comment]
`.DS_Store` needs removing

File: views/patterns/piano/code.html.twig [hunk 1]
```diff
@@ -0,0 +1,279 @@
+{% extends '@pulsar/pulsar/components/tab.html.twig' %}
+{% import 'helpers/example.html.twig' as docs %}
+
+{% block tab_content %}
+<div class="report">
+    <p class="lede">The paino pattern allows you to switch between different panels of content within the same UI. Commonly used in integration and messages UIs.</p>
```
Comment (Stanton) [inline views/patterns/piano/code.html.twig:6]: typo

File: views/patterns/piano/code.html.twig [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        <img src="/views/patterns/piano/img/piano.png" alt="Screenshot showing the layout of an piano UI" />
+    </figure>
+
+    {{ docs.heading('h2', 'Dependencies') }}
+
+    <p>You will need to import the <code>_component.piano.scss</code> sass partial in your main sass file.</p>
+
+    <pre><code lang="sass">@import '/path/to/pulsar/stylesheets/component.piano';</code></pre>
+
+
```
Comment (Stanton) [inline views/patterns/piano/code.html.twig:18]: Double space

File: views/patterns/piano/code.html.twig [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+                        <div class=\"message-body\">
+                            <p>It's nine o'clock on a Saturday
+                            The regular crowd shuffles in<br />
+                            There's an old man sitting next to me
+                            Making love to his tonic and gin</p>
+                        </div>
+                    </div>
+
+                    <form class=\"form piano__form\" method=\"pos\" action=\"/foo\">
```
Comment (Stanton) [inline views/patterns/piano/code.html.twig:228]: post?

**Pull request merged**